.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license
+
+name: Run pre-commit
+
+on:
+- pull_request
+- push
```

Comment:
To avoid running twice on pull request.

```suggestion
- push
    branches:
      - master
```

Reply:
@hartwork 
![image](https://github.com/kevin1024/vcrpy/assets/81854/47c1908f-699c-4e4a-9fea-6ab4f66323ee)

Reply:
@jairhenrique that's actually a feature since the code in isolation and the code merged are not necessarily the same (e.g. when master moved in the mean time) and both states should arguably be tested. I know it's not soemthing that everyone agrees upon. If it means a lot to you I can change it, but I consider this a feature rather than a bug.